util/logging.go

```diff
@@ -36,6 +37,12 @@ const (
 
 	// Name of the environment variable that enables logging of HTTP responses
 	envLogSkipHttpResp = "GOVCD_LOG_SKIP_HTTP_RESP"
+
+	// Name of the environment variable with a custom list of of responses to skip from logging
+	envLogSkipTagList = "GOVCD_LOG_SKIP_TAG_LIST"
+
+	// Name of the environment variable with a custom list of of functions to include in the logging
+	envLogIncludeFunctionList = "GOVCD_LOG_INCLUDE_FUNCTION_LIST"
```

Comment:
Suggestion:
- `GOVCD_LOG_SKIP_TAG_LIST` -> `GOVCD_LOG_SKIP_TAGS`
- `GOVCD_LOG_INCLUDE_FUNCTION_LIST` -> `GOVCD_LOG_FUNCTIONS`
(notice the plural forms too). Open question is whether we go with  "_LOG" or "_API_LOG".

Reply:
done